infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/StreamsSnapshotWriter.java

```diff
@@ -352,6 +320,7 @@ private void applyShadowStream(UUID streamId, long snapshot) {
         if (streamId.equals(REGISTRY_TABLE_ID)) {
             // Only keep entries that currently not exist in registry table
             smrEntries = fetchNewEntries(new ArrayList<>(smrEntries));
+            log.info("{} new entries for registry table found during snapshot sync.", smrEntries.size());
```

Comment:
same here, it will be valuable to log the names of new streams discovered

Reply:
Same to this one: https://github.com/CorfuDB/CorfuDB/pull/3315#discussion_r975950405